Entities/Attachable.cpp

```diff
@@ -247,7 +247,12 @@ bool Attachable::ParticlePenetration(HitData &hd)
 //                  pieces in its place.
 
 void Attachable::GibThis(Vector impactImpulse, float internalBlast, MovableObject *pIgnoreMO) {
-    m_pParent ? m_pParent->RemoveAttachable(this) : Detach();
```

Comment:
Can you undo this change? It doesn't accomplish anything, and I'm working on (i.e. rewriting) attachable stuff so it's gonna add one more little merge conflict for me to deal with.

Reply:
It fails to compile if I don't

Reply:
That's very strange. Why does compilation fail?

Reply:
Yeah it's just one of those things msvc lets you get away with, but gcc/clang don't.

Reply:
Ahh, boo! Alright then, leave it as-is and feel free to resolve this. I'll just push through the merge conflict when it happens.